### ISSUE TITLE
@ldap.group_required not working

### ISSUE THREAD

**Issue by pieterbergmans** [admiralobvious/flask-simpleldap#32]
Hi...I'm trying to restrict a page in my app using @ldap.group_required(groups=['xxx']) but I'm unable to get it working.  I have my group configuration setup like this:

```
# Group configuration
app.config['LDAP_GROUP_MEMBERS_FIELD'] = "uniquemember"
app.config['LDAP_GROUP_OBJECT_FILTER'] = "(&(objectclass=groupOfUniqueNames)(uniquemember=%s))"
app.config['LDAP_GROUP_MEMBER_FILTER'] = "member=%s"
app.config['LDAP_GROUP_MEMBER_FILTER_FIELD'] = "cn"
```

When I print the groups I belong to to the console I get this:

`[b'web', b'confluence', b'google']`

When I restrict a page to a group I'm not associated with but that is one of our ldap groups, nothing happens. This is the code I'm using to restrict my 'dbadmin' page:
```
@app.route('/group')
@ldap.group_required(groups=['dbadmin'])
def group():
    return 'For dbadmins only'
```

By the way, @ldap.login_required works correctly.  Also, I'm using Flask 0.12 and python 3.6.0.

**Comment by pieterbergmans** [comment]
Ok...I figured it out but I'm not sure the best way to handle the fix.  The quick fix was to put a 'b' in front of my group name, like so:

```
@app.route('/group')
@ldap.group_required(groups=[b'web'])
def group():
    return 'For web developers only'
```

...since the list of groups my user account is associated with looks like this (as I mentioned in my original post):

`[b'web', b'confluence', b'google']`

The alternative is to change the `flask_simpleldap/__init__.py` file to decode the `bytes` to a `str` in the `get_user_groups()` method like so (line 235):

```
groups = [record[1][group_member_filter][0].decode('utf-8') for
                              record in records]
```

I'm not a fan of editing the `__init__.py` file but I already had to edit this file to get the `bind_user()` method to work.

**Comment by admiralobvious** [comment]
Hello, indeed, seems like I messed up the py2/3 thing. I'll fix both of those issues.

**Comment by admiralobvious** [comment]
I fixed it in the latest version, 1.1.2 can you check that it's resolved for you?

**Comment by admiralobvious** [comment]
@pieterbergmans I changed the behaviour in 1.2.0 to return strings instead of bytes.
